packages/database/src/core/PersistentConnection.ts

```diff
@@ -948,8 +948,8 @@ export class PersistentConnection extends ServerActions {
 
     // Puts depend on having received the corresponding data update from the server before they complete, so we must
     // make sure to send listens before puts.
-    forEach(this.listens_, (pathString: string, queries: Object) => {
-      forEach(queries, (key: string, listenSpec: ListenSpec) => {
+    forEach(this.listens_, (_, queries) => {
+      forEach(queries, (_, listenSpec) => {
```

Comment:
It means this variable is exempt from unused variable checks (this check is not turned on in our repo right now): https://github.com/Microsoft/TypeScript/issues/9458

It can just start with an underscore though, the rest of the name doesn't matter, so I'll add the names back.

Reply:
👍